README.md

```diff
@@ -12,7 +12,7 @@ steps:
     java-version: '12.x'
 - uses: subosito/flutter-action@v1
   with:
-    flutter-version: '1.9.1+hotfix.6'
+    flutter-version: 'v1.9.1+hotfix.6'
```

Comment:
Thanks for the PR @alexhenriquepv. This change will break existing usage. it would be great if we can keep the current format while handling new versioning.

Reply:
May be upgrade to `flutter-action@v2`? Next stable is around the corner

![image](https://user-images.githubusercontent.com/7928961/79081726-92afb380-7d52-11ea-86f3-19873e98171e.png)

Reply:
@subosito I think we should just version up the action to v2. I've opened a PR with a fix at https://github.com/subosito/flutter-action/pull/35

Reply:
@subosito I think we should just release v2 of the action. I opened a very small PR to fix this at #35 which just uses the input version and doesn't make any assumptions. This will allow for devs to specify the version they want regardless of if the format continues to change in the future.

Reply:
@felangel starting from v1.3.0 we now don't construct download URL manually; instead, we use data from release.archive, https://github.com/subosito/flutter-action/blob/master/src/installer.ts#L190, so it should be safe for now.